src/app/portlets/dot-apps/dot-apps-configuration-detail/dot-apps-configuration-detail-form/dot-apps-configuration-detail-form.component.html

```diff
@@ -16,6 +16,12 @@
                         }"
                         >{{ field.label }}</label
                     >
+                    <dot-icon
```

Comment:
Why don't you create a component so you don't have to repeat the same code 3 times?